packages/devtools_app/lib/src/shared/analytics/analytics_common.dart

```diff
@@ -128,6 +130,20 @@ Map<String, String?> createStackTraceForAnalytics(
   return stackTraceChunks;
 }
 
+/// Normalize a file path from either platform to the POSIX equivalent so that
+/// paths are the same for both Windows and non-Windows paths.
+String _normalizePath(String path) {
+  // Windows path with drive letter.
+  if (!path.startsWith('/') && path.length > 2 && path[1] == ':') {
```

Comment:
nit: we could use a regexp here to ensure that the first chat is a capital letter and the second char is `:`, but idk if the first char could ever be anything other than A-Z.

Reply:
It's not a requirement that the letter is capital, but you're right a regex would've been better. Changed this to `RegExp(r'^[a-zA-Z]:');` which does the same but also ensures a single letter before the colon. Thanks!